code/modules/virus2/predefined.dm

```diff
@@ -128,14 +127,13 @@ var/list/global_diseases = list()
 /datum/disease2/disease/predefined/cult
 	form = "Curse"
 	category = DISEASE_CULT
-	stage_variance = -4
+	stage_variance = 0
```

Comment:
This makes the disease work like Parasites, aka, when someone gets infected by someone whose disease has incubated all the way to Stage 2, the newly infected individual gets stage 2 symptoms right away. This makes the effects of the disease a lot more visible on the station, and since its deadly Stage 4 was removed I think it's fine.